AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3457,6 +3457,7 @@ obozrevatel.com##div[style^="z-index:999999999999999"]
 fasttorrent.ru,fast-torrent.ru#$?#html:has(> body) { padding-top: 0!important; }
 asos.com##header > ._2pMurAl._2gYXe0w
 spiceworks.com##header > div.sui-site-message-container
+perimeterx.com###gatsby-focus-wrapper > div > .open
```

Comment:
it works. but the rule tells nothing about the subject of blocking. I mean that it's not obvious **what** will be blocked. we are trying not to add such rules

there is the link inside this element so extended-css rule would be better instead of this one. can you try to do that by yourself? or you need some help?

Reply:
Ow I see. I would appreciate if you help me.

Reply:
something like this would be fine
```
perimeterx.com#?##gatsby-focus-wrapper > div > aside:has(> p > a[href="/legal/privacy/"])
```

Reply:
Ok thanks. Done.